x/ccv/provider/keeper/migration.go

```diff
@@ -1,172 +0,0 @@
-package keeper
-
-import (
-	"time"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
-	ibctmtypes "github.com/cosmos/ibc-go/v4/modules/light-clients/07-tendermint/types"
-	providertypes "github.com/cosmos/interchain-security/v2/x/ccv/provider/types"
-	ccvtypes "github.com/cosmos/interchain-security/v2/x/ccv/types"
-)
-
-// Migrator is a struct for handling in-place store migrations.
-type Migrator struct {
-	ccvProviderKeeper     Keeper
-	ccvProviderParamSpace paramtypes.Subspace
-}
-
-// NewMigrator returns a new Migrator.
-func NewMigrator(ccvProviderKeeper Keeper, ccvProviderParamSpace paramtypes.Subspace,
-) Migrator {
-	return Migrator{ccvProviderKeeper: ccvProviderKeeper, ccvProviderParamSpace: ccvProviderParamSpace}
-}
-
-// Migratev1Tov2 migrates a provider from v1.0.0 to v2.0.0, and/or consensus version 1 -> 2.
-func (m Migrator) Migratev1Tov2(ctx sdk.Context) error {
```

Comment:
I'm fine deleting the old migration code, but I assume we'll likely need some form of migration for the new way of storing params right?

Reply:
I'll mark this comment as resolved, all the action from our repo has been taken